capstone/src/test/javascript/graphsTest.js

```diff
@@ -0,0 +1,17 @@
+describe('Data Table Construction', function() {
+  it('Will add correct columns/rows to DataTable from JSON', function() {
+    // TODO(ihsan314): Change this when format of JSON object is well-defined
+    const script = document.createElement('script');
+    script.src = 'https://www.gstatic.com/charts/loader.js';
+    script.defer = true;
+
+    window.initMap = function() {
+      const mockTuitionInfo = {};
+      const expectedData = new google.visualization.DataTable();
+      const actualData = new google.visualization.DataTable();
+      populateDataTable(actualData, mockTuitionInfo);
+      expect(actualData).toBe(expectedData);
```

Comment:
actually - i think this `expect` assertion isn't even being invoked since it's in a function. if you put jiberish in here does the test still pass?

Reply:
I'll check this too, maybe I placed the `expect` assertion incorrectly.

Reply:
It's correct that the `expect` wasn't getting invoked, I'll move it out of the callback.